.github/ISSUE_TEMPLATE/payout-request-proposal.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Payout Request Proposal
+about: Raise a payout request from the Paseo bounty
+title: "[PROVIDER] - [Q#] - [2024]"
+labels: payout-request
+assignees: ''
+
+---
+
+Name: 
+
+Contact details (tg/email/element): 
+
+Proponent: 
+
+Requested DOT: 
+
+Proactive / Retroactive payment: 
+
+Description: 
+
+Evidence:
+
+you can use the payout calculator here https://docs.google.com/spreadsheets/d/10R4s5LVsxHnTbQeHVEd1YXkC09SjjQTMm0Cg9BH6lgY/edit?gid=1924769351#gid=1924769351
+
+| Type          | Starting Date | End Date   | QUARTER | Days | $ / Day | $       | DOT AVG      | DOT       |
```

Comment:
I like the table to describe the items but i would remove the fields:

Days, $ Day, $ , DOT AVG and i would just leave the amount of DOT requested.